.github/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# Changes from release 2024/04 to 2024/05
+1. AIRAC (2405) - Jersey (EGJJ) Stand 32 removed - thanks to @lucak1011 (Luca Kulaga)
```

Comment:
```suggestion
1. AIRAC (2405) - Removed stand 32 from Jersey (EGJJ) SMR - thanks to @lucak1011 (Luca Kulaga)
```
Per the style guide, past tense verb should be at the begining of the change description.